src/plugins/intel_cpu/src/transformations/cpu_opset/arm/pass/convert_i32_div.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2020-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "convert_i32_div.hpp"
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/rt_info.hpp>
+
+namespace ov {
+namespace intel_cpu {
+
+ConvertI32Div::ConvertI32Div() {
+    register_matcher(std::make_shared<ngraph::pattern::Matcher>(ngraph::pattern::wrap_type<ngraph::opset1::Divide>(), "ConvertI32Div"),
+         [](ngraph::pattern::Matcher& m) {
+             auto divide = std::dynamic_pointer_cast<ngraph::opset1::Divide>(m.get_match_root());
+             if (!divide) {
+                 return false;
+             }
+             if (divide->get_element_type() != ov::element::i32) {
```

Comment:
I would propose to use `divide->get_element_type().is_integral_number()` instead. We are going to merge I64 native support, so it would be nice to make this transofmration suitable for upcoming change.